.ci/install_go.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+
+clone_tests_repo
+
+pushd "${tests_repo_dir}"
```

Comment:
nit: I guess we could combine the push/run/pop into a single full path run (that is, drop the push/pop and just prepend the `${tests_repo_dir}` to the command.